src/sentry/static/sentry/app/views/settings/components/forms/model.tsx

```diff
@@ -84,7 +86,7 @@ class FormModel {
       this.setInitialData(initialData);
     }
 
-    this.api = new Client();
+    this.api = new Client(options);
```

Comment:
Instead of passing `options`, can we add a specific option key for it? This way we can be explicit about what options need to be passed to `Client`

Reply:
@billyvg I thought about that but I'm not sure we could do that without explicitly passing in the fields we want. That means we'd need to update this code if the options ever changed for `Client`. Maybe that's an acceptable tradeoff though, thoughts?

Reply:
@scefali hmm can we not do something like `type OptionsWithInitial = FormOptions & {apiOptions: ClientOptions} & {initialData?: object};`?

Reply:
@billyvg good call, that's pretty straightfoward